.github/workflows/close-pending.yaml

```diff
@@ -21,10 +21,10 @@ on:
 jobs:
   close-pending:
     name: close-pending
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Any specific reason for using 18.04? I guess 20.04 is also LTS

Reply:
`latest` [is currently 18.04](https://github.com/apache/ozone/runs/2020674273?check_suite_focus=true#step:1:4), but [will be updated to 20.04](https://github.com/actions/virtual-environments/issues/1816) sometime soon.  I don't expect any problem from the update.  But changing to a more specific version lets us make the upgrade when required in a controlled way, instead of getting it unsolicitedly.

Reply:
makes sense